orthoinference/src/main/java/org/reactome/orthoinference/EWASInferrer.java

```diff
@@ -356,27 +336,27 @@ public static void createAlternateReferenceDBInstance(String toSpeciesLong, JSON
 		alternateDbInst.addAttributeValue(accessUrl, altRefDbJSON.get("access"));
 		alternateDbInst.setAttributeValue(_displayName, ((JSONArray) altRefDbJSON.get("dbname")).get(0));
 		alternateDbInst = InstanceUtilities.checkForIdenticalInstances(alternateDbInst);
-		if (altRefDbJSON.get("alt_id") != null) 
+		if (altRefDbJSON.get("alt_id") != null)
 		{
 			altRefDbId = (String) altRefDbJSON.get("alt_id");
 		}
 		altRefDbExists = true;
 	}
-	
-	public static void setAltRefDbToFalse() 
+
+	public static void setAltRefDbToFalse()
 	{
 		altRefDbExists = false;
 	}
-	
+
 	// Sets the species instance for inferEWAS to use
 	public static void setSpeciesInstance(GKInstance speciesInstCopy)
 	{
 		speciesInst = speciesInstCopy;
 	}
-	
-	public static void resetVariables() 
+
+	public static void resetVariables()
 	{
-		homologueMappings = new HashMap<String,String[]>();
+		homologueMappings = new HashMap<>();
 		ensgMappings = new HashMap<String,ArrayList<String>>();
 		referenceGeneProductIdenticals = new HashMap<String,GKInstance>();
 		ewasIdenticals = new HashMap<String,GKInstance>();
```

Comment:
For consistency, can the typing be removed from the right side for these three lines?

Reply:
Yes, intellij has helpfully grayed them out. Is there a way to purge a project of unnecessary code?

Reply:
I haven't tried it myself, but according to Google there is a drop down option for "Code | Inspect Code" that should show things picked up by IntelliJ.  I don't know if there's a way to purge in one step, though.